source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst

```diff
@@ -10,8 +10,8 @@ Initial DNS Seedlist Discovery
 :Authors: Derick Rethans
 :Status: Draft
 :Type: Standards
-:Last Modified: 2019-04-15
-:Version: 1.4.0
+:Last Modified: 2019-09-XX
+:Version: 1.5.0
```

Comment:
Note: I'll rebase this after https://github.com/mongodb/specifications/pull/1067 is merged.